src/routes/youtube/+page.svelte

```diff
@@ -4,81 +4,82 @@
 
   export let data: PageData;
 
-  let videoData = data.payload.videoData;
-  let channelData = data.payload.channelData;
+  const videoData = data.payload.videoData;
+  const channelData = data.payload.channelData;
   let displayedVideos = videoData;
 
   let rerender: boolean = false;
-
-  // Sort channel data by subcount (bubbling Climate Town to the top)
-  function channelSort(a: any, b: any) {
+  
+  interface Channel {
+    channelId: string;
+    channelSubCount: number;
+  };
+  interface ChannelFilterItem { 
+    channelId: string;
+    active: boolean;
+  };
+  interface Video {
+    channelId: string;
+    [key: string]: any;
+  };
```

Comment:
Could we move this into the `interfaces.ts` file?

Reply:
That's a great idea. I'll take care of that 👍